src/nspawn/nspawn-mount.h

```diff
@@ -23,6 +23,14 @@
 
 #include "cgroup-util.h"
 
+typedef enum MountSettingsMask {
+        MOUNT_FATAL             = 1 << 0, /* if set, a mount error are considered fatal */
+        MOUNT_USE_USERNS        = 1 << 1, /* if set, the mounts are patched considering uid/gid shifts in a user namespace */
+        MOUNT_IN_USERNS         = 1 << 2, /* if set, the mount is executed in the inner child, else in the outer child */
+        MOUNT_MASK_NETNS        = 1 << 3, /* masks mounts to be executed in a network namespace in addition to all other mounts */
+        MOUNT_MASK_RO           = 1 << 4, /* masks mounts to be executed read-only in addition to all other mounts */
```

Comment:
i am a bit confused by the latter two. do these flags now turn the relevant mount operations off? or do they turn them on?  the comments need to be clearer about this.

If i got this right, the MASK_RO set means "/proc/sys is read-only", correct? And if MASK_NETNS is set that means "/proc/sys/net" is read-only, right?

Thinking about this I have the suspicion that "mask" is actually a weird word here and we really shouldn't use it. After all, we use these flags both for the "mask" that is set in the env var, but also for the flags in the mount table. And one is a mask, the other a condition i would say...

Maybe we should call them "MOUNT_APPLY_APIVFS_RO" and "MOUNT_APPLY_APIVFS_NETNS" or so... dunno.

sorry for the confusoin around this...